source/NVDAObjects/UIA/__init__.py

```diff
@@ -1832,6 +1832,24 @@ def event_UIA_notification(self, notificationKind=None, notificationProcessing=U
 				speech.cancelSpeech()
 			ui.message(displayString)
 
+	def event_UIA_dragComplete(self):
+		"""
+		Raised by an element when a drag operation on it is completed.
+		A prominent example is rearranging Start screen tiles on Windows 8 and later.
+		"""
+		# #12271: speech does not report new location but braille does.
+		self.reportFocus()

```

Comment:
This puzzles me.

1. Why are you not mimicking gainFocus behavior completely here? Especially for vision, it makes sense to communicate the new location so the highlighter will follow.

Reply:
Oops - looks like a mismatch between this PR and WinTenApps where the latter doees queue a gain focus event. I'll correct this today.